app/controllers/api/abstract_controller.rb

```diff
@@ -11,6 +11,12 @@ class AbstractController < ApplicationController
       sorry "Can't find #{exc.klass}(s) with ID(s): #{exc.params}", 404
     end
 
+    rescue_from Mongoid::Errors::InvalidFind do |exc|
+      sorry "You most likely forgot to provide an `*_id` attribute in your "\
+            "request parameters. Examples of possible missing params: "\
+            "schedule_id, sequence_id, id, etc..", 400
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.